### DISCUSSION TITLE
Cyberduck edit text file

### DISCUSSION THREAD

**Discussion by Supersimmons** [iterate-ch/cyberduck#16103] [Q&A]
Hi, I'm using version 9 on macOS Sonoma, but I've noticed that when I open a text file from my S3 bucket, edit it, and save it, the file on S3 remains original and isn't changed. I'm almost sure that before version 9 the files opened from the bucket and modified were automatically synchronized so as to avoid a manual transfer.
A thousand thanks

**Comment by dkocher** [comment]
I am not able to reproduce any issues editing files in a S3 bucket.

**Reply by Supersimmons** [reply]
good morning, so if you open a text file directly from Cyberduck and edit it, does it automatically update on the S3 bucket too?

**Reply by tatze96** [reply]
I also can't reproduce issues when editing text files in an S3 storage. The changes are uploaded and present within the storage as soon as I save the file.

Which text editor do you use?

**Reply by Supersimmons** [reply]
I'm using TextEdit, the default text editor in my macOS
